sdk/formrecognizer/ai-form-recognizer/src/constants.ts

```diff
@@ -1,11 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * Defines known cloud audiences for Form Recognizer.
+ *
+ * --- More about national clouds ---
+ * 
+ * National clouds are physically isolated instances of Azure.
```

Comment:
Some more comments on this section:
- "physically isolated" could be interpreted to mean an air-gapped or offline system, which isn't what we're referring to here.
- it helps readability if each line is no longer than about 120 characters (I recommend setting up 80 and 120 character rulers in VSCode settings and splitting documentation lines at 120 characters, so that the section is still readable in text views without automatic line breaks (such as GitHub)
- I don't know if we need to describe _what a sovereign cloud is_ so much as _how to use this value_, which your last sentences get at. What about just:

```javascript
/**
 * Defines the known cloud audiences for Form Recognizer.
 *
 * To authenticate with Azure Active Directory (using a `TokenCredential`) in a [Sovereign Cloud](<link here>) 
 * environment, provide the appropriate value below as the `audience` option when creating a 
 * `DocumentAnalysisClient` or `DocumentModelAdministrationClient`.
 *
 * The default value is suitable for Form Recognizer resources created in the Azure Public Cloud, so this value
 * is only required to use Form Recognizer in a different cloud environment.
 */
```